card/mod/core/set/all/type.rb

```diff
@@ -18,6 +18,8 @@ def type_name
   type_card.try :name
 end
 
+alias_method :type, :type_name
```

Comment:
translating json to card attributes failed because that method doesn't exist.
I feel if we support `card = Card.create type: "basic"` then there should be a `card.type`.
If not we have to fix `ensure_card` 

There is also the general pattern that `something_name` returns a name object and`something` a string that we break here